php/Dockerfile-debian

```diff
@@ -4,28 +4,31 @@
 ARG PHP_BASE_IMAGE_VERSION
 FROM php:${PHP_BASE_IMAGE_VERSION}
 
-# Install system packages for PHP extensions recommended for Yii 2.0 Framework
+# Install system packages for PHP extensions recommended for Yii
 COPY --from=mlocati/php-extension-installer /usr/bin/install-php-extensions /usr/local/bin/
+# See https://github.com/mlocati/docker-php-extension-installer#supported-php-extensions
 RUN install-php-extensions \
-        soap \
-        zip \
-        curl \
         bcmath \
         exif \
-        gd \
-        iconv \
+        sockets \
+        zip \
         intl \
-        mbstring \
-        opcache \
+        gettext \
         pdo_mysql \
         pdo_pgsql \
-        imagick \
         mongodb \
+        gd \
+        imagick \
+        opcache \
         xdebug
 
 # Disable xdebug by default (see PHP_ENABLE_XDEBUG)
 RUN rm /usr/local/etc/php/conf.d/docker-php-ext-xdebug.ini
 
+# Install Composer
+RUN install-php-extensions @composer-2.1.3
```

Comment:
@WinterSilence 
Do we need to specify a version here?

Reply:
@WinterSilence my fail - i use required versions and not update when copy